drivers/sensor/si7006/Kconfig

```diff
@@ -0,0 +1,7 @@
+
+
+menuconfig SI7006
```

Comment:
Can make this a plain `config` symbol. `menuconfig` is used when there are symbols immediately after the symbol that depend on it, and you want to display those in a separate menu.

There's an overview of `menuconfig` symbols on the [Kconfig tips and tricks](https://docs.zephyrproject.org/latest/guides/kconfig/index.html#menuconfig-symbols) page.